data-model/SDP-sample-records/person_sample1.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/person/1",
+  "label": "John bar Me'sam (6th c. CE)",
+  "entity_type": "person",
```

Comment:
distinguish between individuals and groups? Different entity type? or a separate boolean field?